tools/install_fedora_deps.sh

```diff
@@ -26,7 +26,7 @@ sudo dnf install -y -q \
     boost-test \
     boost-thread \
     ccache \
-    clang \
+    clang-15 \
```

Comment:
Any reason why clang-15 is required?

Reply:
It's because ubuntu and fedora's clang versions are too old and generated eBPF code that was rejected by the eBPF verifier. `clang-15` was the oldest version that worked with the kernel version we are using for the tests

Reply:
Does the TC back end check the clang version? It should either throw an error or disable itself if clang-15 is not available.

Reply:
I pushed a change for this yesterday, which checks whether we have at least clang-15 (anything above that should also work for the current `p4tc` kernel). However the tests are failing because they are not being able to clone p4c. Can you try to run them again? I don't think I have permissions to do so.